<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chisel</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ActionNew"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="ActionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ActionAbout"/>
    <addaction name="ActionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="ToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionShowGrid"/>
   <addaction name="Action3DGrid"/>
   <addaction name="ActionDecreaseGridSize"/>
   <addaction name="ActionIncreaseGridSize"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionLoadWindowState"/>
   <addaction name="ActionSaveWindowState"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionCarve"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionGroup"/>
   <addaction name="ActionUngroup"/>
   <addaction name="ActionIgnoreGroups"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionHideSelected"/>
   <addaction name="ActionHideUnselected"/>
   <addaction name="ActionShowAllVisgroups"/>
  </widget>
  <widget class="QToolBar" name="toolBar_5">
   <property name="windowTitle">
    <string>toolBar_5</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="ActionCut"/>
   <addaction name="ActionCopy"/>
   <addaction name="ActionPaste"/>
  </widget>
  <widget class="QToolBar" name="toolBar_6">
   <property name="windowTitle">
    <string>toolBar_6</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionToggleCordonState"/>
   <addaction name="actionEditCordonBounds"/>
  </widget>
  <widget class="QToolBar" name="toolBar_7">
   <property name="windowTitle">
    <string>toolBar_7</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionToggleSelectByHandles"/>
   <addaction name="actionToggleAutoSelection"/>
   <addaction name="ActionTextureLock"/>
  </widget>
  <widget class="QToolBar" name="toolBar_8">
   <property name="windowTitle">
    <string>toolBar_8</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionTextureAlignment"/>
  </widget>
  <widget class="QToolBar" name="toolBar_9">
   <property name="windowTitle">
    <string>toolBar_9</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionRunMap"/>
  </widget>
  <action name="ActionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="ActionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
  </action>
  <action name="Action3DGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D Grid</string>
   </property>
  </action>
  <action name="ActionDecreaseGridSize">
   <property name="text">
    <string>Decrease grid size</string>
   </property>
  </action>
  <action name="ActionIncreaseGridSize">
   <property name="text">
    <string>Increase grid size</string>
   </property>
  </action>
  <action name="ActionLoadWindowState">
   <property name="text">
    <string>Load Window State</string>
   </property>
  </action>
  <action name="ActionSaveWindowState">
   <property name="text">
    <string>Save window state</string>
   </property>
  </action>
  <action name="ActionCarve">
   <property name="text">
    <string>Carve</string>
   </property>
  </action>
  <action name="ActionGroup">
   <property name="text">
    <string>Group</string>
   </property>
  </action>
  <action name="ActionUngroup">
   <property name="text">
    <string>Ungroup</string>
   </property>
  </action>
  <action name="ActionIgnoreGroups">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ignore groups</string>
   </property>
  </action>
  <action name="ActionHideSelected">
   <property name="text">
    <string>Hide Selected</string>
   </property>
  </action>
  <action name="ActionHideUnselected">
   <property name="text">
    <string>Hide unselected</string>
   </property>
  </action>
  <action name="ActionShowAllVisgroups">
   <property name="text">
    <string>Show all visgroups</string>
   </property>
  </action>
  <action name="ActionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="ActionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="ActionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="ActionToggleCordonState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle cordon state</string>
   </property>
  </action>
  <action name="actionEditCordonBounds">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit cordon bounds</string>
   </property>
  </action>
  <action name="ActionToggleSelectByHandles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle select-by-handles</string>
   </property>
  </action>
  <action name="actionToggleAutoSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle auto-selection</string>
   </property>
  </action>
  <action name="ActionTextureLock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Texture lock</string>
   </property>
  </action>
  <action name="ActionTextureAlignment">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Texture alignment</string>
   </property>
  </action>
  <action name="ActionRunMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Run Map</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="ActionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="ActionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="ActionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
